dist/s_docs

```diff
@@ -244,17 +255,6 @@ EOF
     fi
 }
 
-# Do not build the documentation if doxygen version is not compatible.
```

Comment:
On machines where aspell and doxygen aren't installed we used to exit with `0`, but now we'll call `spellchk` and return an error: `(echo "Cannot find aspell!"; exit 1)`.

Can you check with a team member who builds on macOS that this doesn't break their s_all?

Reply:
I use MacOS and it doesn't break - probably because I do have aspell.
I didn't touch aspell.
I think, the reason it didn't fail for you is because you don't have neither Docygen nor aspell and the script exited normally upon ont finding Doxygen.
Do you want me to change the no-aspell exit code to 0 too?

Reply:
I checked out wiredtiger on my laptop - so no environment setup - and I get the following from running `s_docs`:
```
ajmorton@M-C02GD1VQMD6M dist % ./s_docs 
Cannot find aspell!
./s_docs: line 135: aspell: command not found
./s_docs skipped: unsupported version of doxygen: 1.9.7, not 1.8.17, 1.9.1, or 1.9.3
ajmorton@M-C02GD1VQMD6M dist % echo $?
0
```
so it looks like the subshell is eating the exit code and contuing with no issue. I think exit code `0` with no subshell will work.

Reply:
Done and changed the message so that it can be parsed by outer scripts.